### ISSUE TITLE
griffe not working

### ISSUE THREAD

**Issue by rectalogic** [felix-martel/pydanclick#31]
First, the docs say install `pydanclick[griffe]` but that fails:
```sh-session
$ pip install pydanclick[griffe]
WARNING: pydanclick 0.3.0 does not provide the extra 'griffe'
```

So I manually install `griffe`, but then `pydanclick` `parse_attribute_documentation` tries to import this:

https://github.com/felix-martel/pydanclick/blob/e0d762f5983804d8716810304db57291a6e5982f/pydanclick/model/docstrings.py#L30-L31

which fails and so griffe is disabled:
```
>>> from griffe.dataclasses import Docstring
Traceback (most recent call last):
  File "<python-input-0>", line 1, in <module>
    from griffe.dataclasses import Docstring
ModuleNotFoundError: No module named 'griffe.dataclasses'
```

I'm guessing an older version of griffe had `griffe.dataclasses` and so the missing `pydanclick[griffe]` extra should constrain the griffe version to the working version?

**Comment by felix-martel** [comment]
Yep, looks like a mistake on my side, I'll look into it. Thanks for the report!

**Comment by rnv812** [comment]
Hello, it seems that this fix is not yet published to PyPI. Could you please publish it?

**Comment by felix-martel** [comment]
@rnv812 I released version `0.4` just now

**Comment by rnv812** [comment]
Thank you!
